case/src/test/kotlin/com/ritense/case/web/rest/CaseTabResourceIntTest.kt

```diff
@@ -47,8 +55,9 @@ class CaseTabResourceIntTest : BaseIntegrationTest() {
     @WithMockUser(username = "user@ritense.com", authorities = [USER])
     fun `should get case tabs`() {
         val caseDefinitionName = "some-case-type"
+        val document = createDocument(caseDefinitionName)
         mockMvc.perform(
-            get("/api/v1/case-definition/{caseDefinitionName}/tab", caseDefinitionName)
+            get("/api/v1/document/{documentId}/tab", document.id.id)
```

Comment:
For as long as we don't remove the old endpoint yet, we should keep the tests for it.

Reply:
added back in